### ISSUE TITLE
Doing the loading thing again

### ISSUE THREAD

**Issue by hamswolo** [yuliskov/SmartYouTubeTV#704]
Needs update doing the loading screen  when backing out of videos. Thank you this app is the best

**Comment by demux4555** [comment]
Yup, YouTube has changed something again, and we're basically back to the same place as we were here... https://github.com/yuliskov/SmartYouTubeTV/issues/676

**Comment by Texasgal1** [comment]
doing the same on mine started 6pm 11-11-2020. suggestions? wont load videos. just spins.

**Comment by Rathna-K** [comment]
Same here @yuliskov getting erro: “URL Doesn’t exists” json == null 
EDIT: Might be a wide-spread outage as per this tweet from youtube: https://twitter.com/teamyoutube/status/1326681978037444608

**Comment by hamswolo** [comment]
> doing the same on mine started 6pm 11-11-2020. suggestions? wont load videos. just spins.

Last time this happened using the LTE version worked. The dev is currently working on a new YouTube app so his attention  is elsewhere. He should get to fixing it soon....hopefully I can't live without this app.

**Comment by Chappelli70** [comment]
> Yup, YouTube has changed something again, and we're basically back to the same place as we were here... #676

Again??? I'm new to the app, obviously these glitches have happened before? Hope they get it sorted. It's my favorite app. Brilliant when working properly.

**Comment by Chappelli70** [comment]
Since Smart YouTube came back online, I'd say 70% of videos now won't load on my Nvidea Shield PRO.

**Comment by hamswolo** [comment]
> Since Smart YouTube came back online, I'd say 70% of videos now won't load on my Nvidea Shield PRO.

They the LTE main im using it and its working fine.

**Comment by hamswolo** [comment]
> > Yup, YouTube has changed something again, and we're basically back to the same place as we were here... #676
> 
> Again??? I'm new to the app, obviously these glitches have happened before? Hope they get it sorted. It's my favorite app. Brilliant when working properly.

The developer  is working on a new version of this app , he used to be on top of these thing immediately.  Now his efforts are on the other app. He should fix it soon. 
For now try the LTE version.  Its working for me the menus are just more like the standard youtube.

**Comment by Chappelli70** [comment]
> > Since Smart YouTube came back online, I'd say 70% of videos now won't load on my Nvidea Shield PRO.
> 
> They the LTE main im using it and its working fine.

Where do I get that?

**Comment by hamswolo** [comment]
> > > Since Smart YouTube came back online, I'd say 70% of videos now won't load on my Nvidea Shield PRO.
> > 
> > 
> > They the LTE main im using it and its working fine.
> 
> Where do I get that?

I have a firetv I have to force stop the app and restart it when you see the smarttv splash screen I push the back button this brings you to a screen to select t the launcher. Choose lite main. 
There was just an update to the app said fixed music videos not playing. I went back to the full app and the loading issue is still there I had to return to the lite version.

**Comment by hamswolo** [comment]
Try smarttubenext. Its in beta its his new youtube app

**Comment by demux4555** [comment]
> They the LTE main im using it and its working fine.

It's still just as broken, unfortunately. 

The developer is pushing the users over on a different app without any explanation or argumentation for why we should now be using a different app... There's no wonder why the users are extremely reluctant to change.  Everyone is very happy with the existing app (when it works, of course). 

Perhaps things would be a lot smoother if the developer would explain what's going on, and why the new app is something we should move over to :)  Also, like others have mentioned, people don't want to move to a new app when the new app is so early in development. The vast majority just want to watch Youtube. They don't want to be beta testers for a new app.

**Comment by hamswolo** [comment]
I'm using the lite version its working for me.

I downloaded the new app its working as well and seems to be  more customizable. Its looks a bit different  but seems like just something  to get used to.

**Comment by Rathna-K** [comment]
> I'm using the lite version its working for me.
> 
> I downloaded the new app its working as well and seems to be more customizable. Its looks a bit different but seems like just something to get used to.

Where can I get the new app? TIA

**Comment by hamswolo** [comment]
https://github.com/yuliskov/SmartTubeNext

**Comment by hamswolo** [comment]
> > I'm using the lite version its working for me.
> > I downloaded the new app its working as well and seems to be more customizable. Its looks a bit different but seems like just something to get used to.
> 
> Where can I get the new app? TIA

https://github.com/yuliskov/SmartTubeNext
